sdk/storage/azfile/file/client_test.go

```diff
@@ -4256,8 +4255,8 @@ func (f *FileUnrecordedTestsSuite) TestFileUploadRangeFromURLNow() {
 	_require.NotNil(cResp.FileLastWriteTime)
 
 	uResp, err := destFClient.UploadRangeFromURL(context.Background(), srcFileSAS, 0, 0, int64(contentSize), &file.UploadRangeFromURLOptions{
-		SourceContentCRC64: contentCRC64,
-		LastWrittenMode:    to.Ptr(file.LastWrittenModeNow),
+		SourceContentValidation: file.SourceContentValidationTypeCRC64(contentCRC64),
```

Comment:
Let's also add a test where `SourceContentValidation` is nil in the `UploadRangeFromURLOptions`.

Reply:
Added